proto/cmp/types/v1alpha/rate.proto

```diff
@@ -26,11 +26,55 @@ enum RateRuleType {
 message RatePlan {
```

Comment:
Comment for `RatePlan` should be updated according to new changes.

Reply:
multiple comments in rate.proto revised